features/fetch-crl/config.pan

```diff
@@ -11,31 +15,45 @@ variable SITE_DEF_CERTDIR      ?= SITE_DEF_GRIDSEC_ROOT+"/certificates";
 
 # Include RPMs
 variable RPMS_CONFIG_SUFFIX ?= '';
-include { 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX };
+include 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX;
 
 # Define fetch-crl version if not defined when adding RPMs
 variable FETCH_CRL_VERSION ?= '3.0';
 
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
 # fetch-crl configuration
-# ----------------------------------------------------------------------------
-include { 'components/sysconfig/config' };
-"/software/components/sysconfig/files/fetch-crl/CRLDIR" = SITE_DEF_CERTDIR;
-"/software/components/sysconfig/files/fetch-crl/FORCE_OVERWRITE" = if ( FETCH_CRL_FORCE_OVERWRITE ) {
-                                                                     'yes';
-                                                                   } else {
-                                                                     'no';
-                                                                   };
-"/software/components/sysconfig/files/fetch-crl/QUIET" = if ( FETCH_CRL_QUIET ) {
-                                                           'yes';
-                                                         } else {
-                                                           'no';
-                                                         };
-
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
+include 'components/metaconfig/config';
+include 'features/fetch-crl/schema';
+# property 'daemons' and 'convert/yesno' are not supported prior to version 16.6. Additionally,
+# in version of metaconfig before 15.12, metaconfig doesn't publish its version.
+# Exit with an error if an older version is used.
+prefix '/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}';
+variable CHECK_VERSION = if ( !exists('/software/components/metaconfig/version') ||
+                              ( (value('/software/components/metaconfig/version') <= '16.6') &&
+                                (!match(value('/software/components/metaconfig/version'), '^16.1')) )  ) {
```

Comment:
16.1 doesn't exist, just everything starting by 16.1 (16.10 and 16.12 probably) should be considered ok.